spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/topic-resolution.adoc

```diff
@@ -48,6 +48,20 @@ record Foo(String value) {
 }
 ----
 
+Property placeholders and SpEL expressions are supported in the `@PulsarMessage` annotation,
+for example:
+[source,java,indent=0,subs="verbatim"]
+----
+@PulsarMessage(topic = "${app.topics.foo}")
+record Foo(String value) {
+}
+----
+
+WARNING: Property placeholders and SpEL expressions are by default supported within a Spring Boot context.
```

Comment:
As long as the DTR is a Spring bean (even if a non-SB app manually configured it) the BFA will get called and it will resolve expressions as expected. In fact, once we add a couple of tests to the `Reactive/PulsarListener/ReaderTests` they do not use SB but just provide a `@Configuration` w/ the DTR `@Bean` manually wired up.

I think my comment got mixed up earlier about this, the part that you get from using SB is when the default topic mappings (`spring.pulsar.defaults.type-mappings` or `spring.pulsar.consumer.topic` or `spring.pulsar.producer.topic`) have expressions, the consumption of these by SB from the yml files will do the resolution for us. W/o Boot there are no yml and the mappings would be added manually by the users etc..

I also think this 2nd fact is a detail that existed before this change and it might be better to just leave it out to avoid confusion.